lib/kitchen/driver/vagrant.rb

```diff
@@ -389,6 +402,12 @@ def update_state(state)
         state[:rdp_port] = hash["RDPPort"] if hash["RDPPort"]
       end
 
+      # @return [String] full absolute path to the kitchen cache directory
+      # @api private
+      def local_kitchen_cache
+        @local_kitchen_cache ||= File.expand_path('~/.kitchen/cache')
```

Comment:
The unfortunate issue with this path is that it makes this cache cookbook local. A user may test many cookbooks (very common) and each one will need to maintain their own cache in its local `.kitchen` folder. Would be nice to move it to a temp directory of the host.